groups/ecs-service/data.tf

```diff
@@ -29,3 +29,18 @@ data "aws_ssm_parameter" "secret" {
   for_each = toset(data.aws_ssm_parameters_by_path.secrets.names)
   name     = each.key
 }
+
+# retrieve all global secrets for this env using global path
+data "aws_ssm_parameters_by_path" "global-secrets" {
+  path = "/${local.global_prefix}"
+}
+# create a list of secrets names to retrieve them in a nicer format and lookup each secret by name
+data "aws_ssm_parameter" "global-secret" {
+  for_each = toset(data.aws_ssm_parameters_by_path.global-secrets.names)
+  name     = each.key
+}
```

Comment:
We need to be careful with our consistency here when naming the resources / data lookups. Ideally we'd be using underscores, rather than hyphens.

E.g. `global-secrets` would ideally be `global_secrets`